docs/known-issues.md

```diff
@@ -0,0 +1,8 @@
+# Known Issues
```

Comment:
Great idea to add the known issues page. 
I think you should remove that first sentence. The subordinate clause contains information that is not readily understood by customers. For example, *workload teams* may not be widely understood. 
Instead, I suggest that we point out that the client library: 1) is a snapshot of the Microsoft Graph and may not be in sync with the service at any one point so issues may arise from that, 2) the client library is generated; the generator may not capture all potential scenarios, 3) service behavior changes may not be reflected in the client library as the development of the client library and the Microsoft Graph are loosely coupled